content/docs/next/op-guide/runtime-configuration.md

```diff
@@ -241,12 +241,12 @@ It is enabled by default.
 [conf-adv-peer]: configuration.md#clustering-flags
 [conf-name]: configuration.md#member-flags
 [disaster recovery]: recovery.md
-[fault tolerance table]: ../../v2/admin_guide.md#fault-tolerance-table
 [majority failure]: #restart-cluster-from-majority-failure
-[member-api]: ../../v2/members_api.md
 [member-api-grpc]: ../dev-guide/api_reference_v3.md
-[member migration]: ../../v2/admin_guide.md#member-migration
 [remove member]: #remove-a-member
 [runtime-reconf]: runtime-reconf-design.md
 [error cases when promoting a member]: #error-cases-when-promoting-a-learner-member
 [design-learner]: ../learning/design-learner.md
+[member-api]: /docs/v2.3/members_api.md
+[member migration]: /docs/v2.3/admin_guide.md#member-migration
+[fault tolerance table]: /docs/v2.3/admin_guide.md#fault-tolerance-table
```

Comment:
Thanks for fixing the links.
Nit: sort these link definitions (using your editor's "sort lines" features). This can be done now or later.

Reply:
I did this in a couple places to shorten the work -- i'd fix one version's page, then copy three lines at once, and paste them into the other versions as needed.

I'm happy to go back through and sort. (will look into the sort lines function)

Reply:
I think the sorting should be left to another PR.

Reply:
Let's track this in issue https://github.com/etcd-io/website/issues/183